### ISSUE TITLE
[BUG] warp bug

### ISSUE THREAD

**Issue by matejkar** [tsunamayo/Starship-EVO#3112]
not sure how to describe this one, and i unfortunately dont have any pics or output logs currently.
Basically, when trying to enter warp speed, the ship would enter a speed of 71Mm/s (always that speed, no matter the ship) and start slowly drifting away from the star/system i targeted. Also it seems like the star lineup changes in the exact frame the ship enters warp, and the targeted system suddenly shows up in the place of the main yoke i pilot the ship from, also when leaving warp the ships maintains the speed of 71Mm/s and if i leave it the ship teleports away. Also there are no line effects like in correctly working warp.

If i manage to get this bug again, i will post screenshots and the output log

**Comment by matejkar** [comment]
update: it seems to happen when leaving the autopilot for warp

**Comment by matejkar** [comment]
[Uploading output_log.txt…]()

**Comment by matejkar** [comment]
[output_log.txt](https://github.com/tsunamayo/Starship-EVO/files/5432886/output_log.txt)

**Comment by matejkar** [comment]
![image](https://user-images.githubusercontent.com/69653738/97082099-6fba6200-1607-11eb-9dc4-498ff43526c2.png)
what i would also see is the line for the warp jump on a completely wrong position, like one of these drawn ones

**Comment by matejkar** [comment]
![20201024143851_1](https://user-images.githubusercontent.com/69653738/97082116-95476b80-1607-11eb-87df-907b3d22122d.jpg)
this is what the bug looks like, 71Mm/s but no warp effect

**Comment by matejkar** [comment]
and the system does not get closer, visually or ingame it still shows the same distance

**Comment by matejkar** [comment]
now after leaving the ship and spawning another, it worked fine again

**Comment by matejkar** [comment]
![20201024152343_1](https://user-images.githubusercontent.com/69653738/97082851-f0c82800-160c-11eb-89cd-fc3f7fcb8d54.jpg)
there we go, happened again now out of nowhere, the selected system is now where the main yoke is, also notice that the fps dropped massively, its between 10 and 25

**Comment by matejkar** [comment]
now its below 10

**Comment by tsunamayo** [comment]
Hi, thanks for booking this.
Now I having a hard time understanding your bug report.
Do you know the step to do to reproduce the issue? ie:
1) Enter yoke
2) Pick warp destination
3) warp
4) ect...
Thanks

**Comment by matejkar** [comment]
once again, its quite inconsistent, but try this:
1. enter yoke
2. pick warp destination
3. initiate jump and while in warp press X to disable autopilot and the ship should be continuing to travel at warp speed
4. after leaving the ship, making it teleport away, spawn a new one, repeat steps 1 and 2 and watch as the warp glitches out like on the pics above

**Comment by CurioInventorium** [comment]
I've had this as well, but it's difficult to explain. For me it was after #3110 happened and I was dumped into deep space after reloading the game and nothing was there.
I spawned a ship to fly to the nearest star system
- entered the yoke
- chose a star system
- then when I initiated the jump, instead of the warp effect the autopilot started moving about erratically and the speed went to Mm/s (I can't remember the value).
- I had a switch attached to the warp drive at the back of the ship, so I thought I could get up from the yoke and head to the back of the ship to shut down warp.
Unfortunately, even with gravity blocks holding me in place, the ship was moving that fast I clipped out of it and was left in space again.

I'll see if this behaviour occurs with a fresh save game, as leaving the starter system with player buildings and ships in place may be a trigger for this bug. Difficult to tell at the moment.
